### DISCUSSION TITLE
Export to ONNX - fail to load converted model with ONNX

### DISCUSSION THREAD

**Discussion by GotG** [Lightning-AI/pytorch-lightning#10489] [code help: CV]
Hello, I am trying to convert a model from https://github.com/hmorimitsu/ptlflow  - FlownetS
Model export is successful (only get the warning below).
However I do not know how to run inference with Onnxruntime.
Notebook:

https://colab.research.google.com/drive/11WtAyF6Rb5E6NaTQa3IqydUQXBEYsZQL?usp=sharing

```
RuntimeError                              Traceback (most recent call last)

<ipython-input-17-a90b84a2f934> in <module>()
----> 1 ort_outs = ort_session.run(None, ort_inputs)[0]

/usr/local/lib/python3.7/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py in run(self, output_names, input_feed, run_options)
    186             output_names = [output.name for output in self._outputs_meta]
    187         try:
--> 188             return self._sess.run(output_names, input_feed, run_options)
    189         except C.EPFail as err:
    190             if self._enable_fallback:

RuntimeError: Input must be a list of dictionaries or a single numpy array for input '0'.
```

**Comment by rohitgr7** [comment]
can you share the error? or a minimal reproducible script to check the issue?

**Reply by GotG** [reply]
I think I know the issue - the inputs to an onnx model must be a numpy array, not a torch tensor. 
It seems to work. The output is slightly different from the pytorch model, so looking into that.
